core/vec3.h

```diff
@@ -2,23 +2,32 @@
 #define VEC3_H
 
 #include "constants.h"
+
+#include <ctime>
+#include <exception>
 #include <initializer_list>
 #include <iostream>
+#include <random>
+#include <sstream>
 
 namespace core {
 
 class Vec3 {
+public:
+  static const Vec3 kVecZero; // Vec3(0, 0, 0)
```

Comment:
Vec3 Vec Vec.

How about simply: kZero?

Also drop the comments, it is obvious that zero is zero.